src/components/molecules/Footer.tsx

```diff
@@ -0,0 +1,15 @@
+import SocialMedia from '../atoms/SocialMedia';
+
+const Footer = () => {
+  return (
+    <footer className="flex flex-col absolute w-full justify-center items-center gap-4 bottom-0 px-3 py-14 bg-green">
```

Comment:
please, simplify the styles that are being used. Many of them are not necessary

Reply:
I dont know if `bottom-0` is necessary, but I think yes